app/helpers/events_helper.rb

```diff
@@ -29,16 +30,26 @@ def check_url(id)
     if id
       "/events/#{id}"
     else
-      '/events/loading'
+      "/events/loading"
     end
   end
 
   def getMap(map_url)
     if map_url.nil? || map_url.strip.empty?
-      new_map = ("https://maps.google.com/maps/place?q=Lagos,+Nigeria&ftid=0x103b8b2ae68280c1:0xdc9e87a367c3d9cb") + "&output=embed"
+      new_map = ("https://goo.gl/ULMKDn") + "&output=embed"
     else
       new_map = map_url + "&output=embed"
     end
     new_map
   end
+
+  def event_price(tickets, event)
+    if user_is_attending_event(event)
+      "Attending"
+    elsif tickets.max  == tickets.min 
```

Comment:
Operator `==` should be surrounded with a single space.<br>Trailing whitespace detected.